src/index/views.rs

```diff
@@ -1,4 +1,19 @@
 use crate::prelude::*;
+use pgrx::error;
+use pgrx::pg_sys::{AsPgCStr, RelnameGetRelid};
+
+#[pgrx::pg_extern(volatile, strict)]
+fn _vectors_index_setting(index: String, key: String, value: String) {
+    unsafe {
+        let oid = RelnameGetRelid(index.as_pg_cstr());
+        let id = Handle::from_sys(oid);
```

Comment:
Currently `handle` is smgr id. It should not work.

If we move to `oid`, we can just accept oid. Users use `alter_vector_index('vector_idx_1'::oid, ...)`.